pkg/bufferdecoder/eventsreader.go

```diff
@@ -346,7 +348,10 @@ func readStringFromBuff(ebpfMsgDecoder *EbpfDecoder) (string, error) {
 	res, err := ReadByteSliceFromBuff(ebpfMsgDecoder, int(size-1)) //last byte is string terminating null
 	defer func() {
 		var dummy int8
-		ebpfMsgDecoder.DecodeInt8(&dummy) // discard last byte which is string terminating null
+		err := ebpfMsgDecoder.DecodeInt8(&dummy) // discard last byte which is string terminating null
+		if err != nil {
+			logger.Error("Trying to discard last byte", "error", err)
```

Comment:
I think this should be a debug or warn level, as it is not breaking the string.
If there is a real error here then it will come up with next argument or event.

Reply:
Warn then.